pkg/controller/networkpolicy/crd_utils.go

```diff
@@ -131,6 +140,31 @@ func (n *NetworkPolicyController) createAddressGroupForCRD(peer secv1alpha1.Netw
 	return normalizedUID
 }
 
+// createAddressGroupForClusterGroupCRD creates an AddressGroup object corresponding to a
+// ClusterGroup spec. If the AddressGroup already exists, it returns the key
+// otherwise it copies the ClusterGroup CRD contents to an AddressGroup resource and returns
+// its key. If the corresponding internal Group is not found return empty.
+func (n *NetworkPolicyController) createAddressGroupForClusterGroupCRD(intGrp *antreatypes.Group) string {
+	key, err := store.GroupKeyFunc(intGrp)
+	if err != nil {
+		return ""
+	}
+	// Check to see if the AddressGroup already exists
+	_, found, _ := n.addressGroupStore.Get(key)
+	if found {
+		return key
+	}
+	// Create an AddressGroup object for this Cluster Group.
+	addressGroup := &antreatypes.AddressGroup{
+		UID:          types.UID(key),
```

Comment:
The UID will be "name" of Group?

Reply:
updated and added comment to AddressGroup type